### ISSUE TITLE
Out of my mind

### ISSUE THREAD

**Issue by scytheling** [Wynncraft/Issues#8776]
Died in the boss part of "out of my mind" once i completed the puzzle again the door didn't open.

**Comment by thecoolkogut4** [comment]
Have you tried switch worlds?

**Comment by scytheling** [comment]
I tried switching worlds but that didn't work. I fixed it by talking to the
NPC at the start of the house and doing the quest over again.


On Aug 14, 2017 5:07 PM, "TheDarkSoul" <notifications@github.com> wrote:

Have you tried switch worlds?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Wynncraft/Issues/issues/8776#issuecomment-322323244>,
or mute the thread
